src/screens/OtherProfile/index.js

```diff
@@ -444,7 +537,7 @@ const OtherProfile = () => {
       dispatch
     );
 
-    refetchFeeds();
+    getOtherFeeds();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getOtherFeeds` function is called after voting on a post. Ensure that this function correctly updates the state and UI to reflect the change in vote status.
```diff
+ getOtherFeeds();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->